pkg/env/sensor.go

```diff
@@ -22,4 +22,7 @@ var (
 
 	// EventPipelineOutputQueueSize is used to specify the size of the eventPipeline's output queue
 	EventPipelineOutputQueueSize = RegisterIntegerSetting("ROX_EVENT_PIPELINE_OUTPUT_QUEUE_SIZE", 100)
+
+	// ResyncDisabled is used to specify if Sensor should have re-sync disabled or not
```

Comment:
IMO I think the message used in the FF was a bit clearer. 
```suggestion
	// ResyncDisabled disables the resync behavior of the kubernetes listeners in sensor
```